src/core/instance/lifecycle.ts

```diff
@@ -330,7 +330,7 @@ export function updateChildComponent(
     const propKeys = vm.$options._propKeys || []
     for (let i = 0; i < propKeys.length; i++) {
       const key = propKeys[i]
-      const propOptions: any = vm.$options.props // wtf flow?
+      const propOptions: any = vm.$options.props // flow?
```

Comment:
```suggestion
      const propOptions: any = vm.$options.props
```

Since we're no longer using flow.